keyboards/mechmini/v1/keymaps/pitty/keymap.c

```diff
@@ -0,0 +1,56 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "mechmini.h"
+#include "quantum.h"
+
+#define _QWERTY 0
+#define _NMBR 1
+#define _NAV 2
+#define _MOUSE 3
+#define _FUNCT 4
+
+#define _____ KC_TRNS
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [_QWERTY] = SPLITSPACE_KEYMAP(
```

Comment:
Please change all instances of `SPLITSPACE_KEYMAP` to `LAYOUT_split_space`. (Lines 26, 32, 38, 44, and 50.)

The desired terminology changed a while back and this usage is outdated.